### ISSUE TITLE
cannot install libarchive on windows

### ISSUE THREAD

**Issue by ReenigneArcher** [dsoprea/PyEasyArchive#47]
Hello. I read #27 and followed the guide in the comments - https://github.com/dsoprea/PyEasyArchive/issues/27#issuecomment-574887181

Everything in the guide was accurate except for `libnettle-7.dll` was actually named `libnettle-8.dll`. I don't know that this difference would cause the issue or not.

So I try to install and get a message that libarchive.dll or it's dependencies are missing. libarchive.dll definately exists in the directory it's looking in.

```
(venv) C:\Users\ReenigneArcher\Documents\GitHub\RetroArcher.X>set Path=%PATH%;C:\Users\ReenigneArcher\Documents\GitH
ub\RetroArcher.X\lib\libarchive-3.5.1\bin

(venv) C:\Users\ReenigneArcher\Documents\GitHub\RetroArcher.X>pip install libarchive
Collecting libarchive
  Using cached libarchive-0.4.7.tar.gz (23 kB)
Requirement already satisfied: nose in c:\users\reenignearcher\documents\github\retroarcher.x\venv\lib\site-packages
 (from libarchive) (1.3.7)
Using legacy 'setup.py install' for libarchive, since package 'wheel' is not installed.
Installing collected packages: libarchive
    Running setup.py install for libarchive ... error
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\reenignearcher\documents\github\retroarcher.x\venv\scripts\python.exe' -u -c 'import io, os,
 sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\ReenigneArcher\\AppData\\Local\\Temp\\pip-install-mq647264
\\libarchive_17b4382c4f6549b5a8ce70d3cc464a76\\setup.py'"'"'; __file__='"'"'C:\\Users\\ReenigneArcher\\AppData\\Loca
l\\Temp\\pip-install-mq647264\\libarchive_17b4382c4f6549b5a8ce70d3cc464a76\\setup.py'"'"';f = getattr(tokenize, '"'"
'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'
"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))'
 install --record 'C:\Users\ReenigneArcher\AppData\Local\Temp\pip-record-4f9t6itb\install-record.txt' --single-versi
on-externally-managed --compile --install-headers 'c:\users\reenignearcher\documents\github\retroarcher.x\venv\inclu
de\site\python3.9\libarchive'
         cwd: C:\Users\ReenigneArcher\AppData\Local\Temp\pip-install-mq647264\libarchive_17b4382c4f6549b5a8ce70d3cc4
64a76\
    Complete output (4 lines):
    running install
    error: Could not find module 'C:\Users\ReenigneArcher\Documents\GitHub\RetroArcher.X\lib\libarchive-3.5.1\bin\li
barchive.dll' (or one of its dependencies). Try using the full path with constructor syntax.
    Verifying that the library is accessible.
    Library can not be loaded: Could not find module 'C:\Users\ReenigneArcher\Documents\GitHub\RetroArcher.X\lib\lib
archive-3.5.1\bin\libarchive.dll' (or one of its dependencies). Try using the full path with constructor syntax.
    ----------------------------------------
ERROR: Command errored out with exit status 1: 'c:\users\reenignearcher\documents\github\retroarcher.x\venv\scripts\
python.exe' -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\ReenigneArcher\\AppData\\
Local\\Temp\\pip-install-mq647264\\libarchive_17b4382c4f6549b5a8ce70d3cc464a76\\setup.py'"'"'; __file__='"'"'C:\\Use
rs\\ReenigneArcher\\AppData\\Local\\Temp\\pip-install-mq647264\\libarchive_17b4382c4f6549b5a8ce70d3cc464a76\\setup.p
y'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from
setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(
code, __file__, '"'"'exec'"'"'))' install --record 'C:\Users\ReenigneArcher\AppData\Local\Temp\pip-record-4f9t6itb\i
nstall-record.txt' --single-version-externally-managed --compile --install-headers 'c:\users\reenignearcher\document
s\github\retroarcher.x\venv\include\site\python3.9\libarchive' Check the logs for full command output.
```

TLDR; version
```
    Complete output (4 lines):
    running install
    error: Could not find module 'C:\Users\ReenigneArcher\Documents\GitHub\RetroArcher.X\lib\libarchive-3.5.1\bin\li
barchive.dll' (or one of its dependencies). Try using the full path with constructor syntax.
    Verifying that the library is accessible.
    Library can not be loaded: Could not find module 'C:\Users\ReenigneArcher\Documents\GitHub\RetroArcher.X\lib\lib
archive-3.5.1\bin\libarchive.dll' (or one of its dependencies). Try using the full path with constructor syntax.
```

![image](https://user-images.githubusercontent.com/42013603/127951278-e68e1416-93eb-4fac-9014-f66f6aada05b.png)

**Comment by ganego** [comment]
There is a very easy solution. Just uninstall this library. It's a shame the author no longer cares about it but is active on other repos.

Use https://github.com/Changaco/python-libarchive-c instead. This is the one I've been using for some years now.
